Dockerfile

```diff
@@ -4,5 +4,5 @@ ENV GIT_URL https://github.com/obsidiansystems/kadena-transfer-js.git
 
 RUN apt-get -y update \
 && apt-get -y install git \
-&& git clone -b hw-ledger-signing ${GIT_URL} \
+&& git clone -b ago-ui-fix ${GIT_URL} \
```

Comment:
Should this actually point to `master` or some other fixed branch?

Reply:
Yes, master. I was using this to save time.